diff --git a/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/tryInstallAppOnDevice.js b/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/tryInstallAppOnDevice.js
index 790de0d..4c13059 100644
--- a/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/tryInstallAppOnDevice.js
+++ b/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/tryInstallAppOnDevice.js
@@ -38,7 +38,7 @@ function tryInstallAppOnDevice(args, adbPath, device, androidProject, selectedTa
     const defaultVariant = (args.mode || 'debug').toLowerCase();
 
     // handle if selected task from interactive mode includes build flavour as well, eg. installProductionDebug should create ['production','debug'] array
-    const variantFromSelectedTask = selectedTask === null || selectedTask === void 0 ? void 0 : selectedTask.replace('install', '').split(/(?=[A-Z])/);
+    const variantFromSelectedTask = selectedTask === null || selectedTask === void 0 ? args.variant ? args.variant.split(/(?=[A-Z])/) : void 0 : selectedTask.replace('install', '').split(/(?=[A-Z])/);
 
     // create path to output file, eg. `production/debug`
     const variantPath = (variantFromSelectedTask === null || variantFromSelectedTask === void 0 ? void 0 : (_variantFromSelectedT = variantFromSelectedTask.join('/')) === null || _variantFromSelectedT === void 0 ? void 0 : _variantFromSelectedT.toLowerCase()) ?? defaultVariant;
