diff --git a/node_modules/@react-native-firebase/messaging/lib/index.d.ts b/node_modules/@react-native-firebase/messaging/lib/index.d.ts
index 56d100b..77e5bf8 100644
--- a/node_modules/@react-native-firebase/messaging/lib/index.d.ts
+++ b/node_modules/@react-native-firebase/messaging/lib/index.d.ts
@@ -108,7 +108,7 @@ export namespace FirebaseMessagingTypes {
     /**
      * Any additional data sent with the message.
      */
-    data?: { [key: string]: string };
+    data?: { [key: string]: any };
 
     /**
      * Additional NotificationPayload data sent with the message
