diff --git a/node_modules/tiamut/dist/types.d.ts b/node_modules/tiamut/dist/types.d.ts
index 87d80a8..cfbe7a1 100644
--- a/node_modules/tiamut/dist/types.d.ts
+++ b/node_modules/tiamut/dist/types.d.ts
@@ -1,4 +1,4 @@
-export declare type DropFirst<T extends unknown[]> = T extends [any, ...infer U] ? U : never;
+export declare type DropFirst<T extends unknown[]> = T extends [any, ...infer U] ? U : any;
 export declare type Listener<T, K> = (current: T, previous: T, actionName: K | 'setState' | 'resetState') => void;
 export declare type Action<T> = (state: T, ...params: any[]) => T;
 export declare type ActionsMap<T> = {
